front/src/api/books.js

```diff
@@ -0,0 +1,8 @@
+import { config } from "../config";
+
+export const books = {
+  fetchBooks: async (pageNumber) => {
```

Comment:
W sumie dobra okazja, zebys sie juz zapoznal z ta libka
https://github.com/axios/axios
Sam uzywam do projektow, upraszcza korzystanie z api zamiast zwyklym fetchem.

Po fragmencie "// Want to use async/await? Add the `async` keyword to your outer function/method."

Reply:
Odpowiedz do fragmentu: "Yes! I really do :)"
Done.

Reply:
Done.